gitpod-io/gitpod
# [dashboard] add location for ide_configuration_changed track event

## Description

Add `location` for `ide_configuration_changed` tracking event to make data of experiment more clear

## Related Issue(s)

Fixes #9791

## How to test


Go to preview environment to select IDE, and see if [segment](https://app.segment.com/gitpod/sources/staging_untrusted/debugger) got tracking data with properly `location`

1. Workspace start (e.g. a path with a context URL /#https://github.com/person/project. `location=workspace_start`
2. Dashboard, currently on the workspaces list page. `location=workspace_list`
3. Preferences page (/preferences) as before. `location=preferences `

## Release Notes

```release-note
NONE
```

## Documentation
